homeassistant/components/config/config_entries.py

```diff
@@ -145,6 +146,7 @@ class ConfigManagerFlowIndexView(FlowManagerIndexView):
 
     url = "/api/config/config_entries/flow"
     name = "api:config:config_entries:flow"
+    _data: dict[str, Any] = {}
```

Comment:
I don't think this is OK, because there's a single instance of the `ConfigManagerFlowIndexView` class handling all requests.